dali/driver/hasseb_firmware/README.md

```diff
@@ -0,0 +1,215 @@
+hasseb USB DALI Master firmware
+===============================
+
+## Updating the firmware
+
+The firmware of hasseb USB DALI Master can be updated through USB. By booting
+the device with jumper JP1 short circuited, the device appears as a mass
+storage to your computer. The firmware can be update by removing the old
+firmware.bin file from the mass storage and copying the new firmware to the
+device. The name of the firmware file does not need to be changed. The most
+resent firmware can be found in this folder.
+
```

Comment:
Can we revert to the old firmware by backing up the existing file ?

Reply:
Yes. Old firmware can be found from www.hasseb.fi/dali as well

Reply:
Can you provide a changes list ?

I'm really excited about this :) but my setup has been working flawlessly for 2 years and don't won to mess with that...

Reply:
No I can't, there are too many fundamental changes. The new firmware should be fully documented in the firmware readme file.

Reply:
Under Linux, you can't reliably write the firmware by copying the new firmware to the device.  You need to access the block device directly.  See https://github.com/sde1000/hasseb-usb/blob/master/scripts/flash.sh for a script that does this (although you will need to edit the script to set the firmware filename).

Reply:
You are right. I need to clarify this somehow.

Reply:
Instructions to update drivers in Linux added to readme.md